tests/foreman/installer/test_infoblox.py

```diff
@@ -16,6 +16,117 @@
 """
 import pytest
 
+from robottelo.helpers import InstallerCommand
+
+
+def register_satellite(sat):
+    sat_version = str(sat.execute('rpm -q satellite | cut -d- -f2 | cut -d. -f1,2 | tr -d "\n"'))
+    sat.execute(
+        'yum -y localinstall '
+        'http://dogfood.sat.engineering.redhat.com/pub/katello-ca-consumer-latest.noarch.rpm'
+    )
+    sat.execute(
+        'subscription-manager register --org Sat6-CI '
+        f'--activationkey satellite-{sat_version}-qa-rhel7'
+    )
+    return sat
+
+
+@pytest.mark.tier4
+def test_isc_dhcp_plugin_installation(satellite_latest):
+    """Check that there are no packaging issues with ISC DHCP plugin
+
+    :id: 6fc3827b-e431-4105-b2e9-f302044bdc09
+
+    :Steps: Run installer with option
+        --enable-foreman-proxy-plugin-dhcp-remote-isc
+
+    :expectedresults: Plugin installs successfully
+
+    :CaseAutomation: Automated
+
+    :Requirement: Installer
+
+    :customerscenario: true
+
+    :BZ: 1994490
+    """
+    register_satellite(satellite_latest)
+    installer_obj = InstallerCommand('enable-foreman-proxy-plugin-dhcp-remote-isc')
```

Comment:
I think we can include all these test cases inside the destructive marker.

Reply:
Thanks for the feedback, @devendra104 ! To do this, do I need to do anything other than add the @pytest.mark.destructive decorator to the tests? Should I use the destructive_sat fixture instead of the satellite_latest fixture? Additionally, do we have any documentation on the destructive marker? I wasn't able to find anything by searching in Confluence.

Reply:
yes, you are right, you need to add the `@pytest.mark.destructive` marker and use `destructive_sat` for these test cases.